packages/backend-tasks/src/tasks/TaskWorker.test.ts

```diff
@@ -308,6 +344,7 @@ describe('TaskWorker', () => {
 
       const before = fn1.mock.calls.length;
       await promise2;
+      await new Promise(resolve => setTimeout(resolve, 350));
```

Comment:
I added this to ensure that the "initialDelayDuration" elapsed before being called. This was due to it seemingly passing on sqlite, but not Postgres. I wonder if this breaks the desire of the test, but it seems by the way the test was written, it expects the second task to be called immediately? SO I was curious about the test in general.